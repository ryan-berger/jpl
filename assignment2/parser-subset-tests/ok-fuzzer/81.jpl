show L()
write image .1 to ""
print ""
read image "@r" to N
print ""
