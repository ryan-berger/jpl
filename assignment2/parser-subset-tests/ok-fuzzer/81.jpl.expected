(ShowCmd (CallExpr L))
(WriteImageCmd (FloatExpr 0) "")
(PrintCmd "")
(ReadImageCmd "@r" (VarArgument N))
(PrintCmd "")
